.nextmv/golden/demand-forecasting-ortools/main_test.go

```diff
@@ -31,6 +31,14 @@ func TestGolden(t *testing.T) {
 					Replacement: golden.StableFloat,
 				},
 			},
+			OutputProcessConfig: golden.OutputProcessConfig{
+				RoundingConfig: []golden.RoundingConfig{
+					{
+						Key:       ".solutions[].forecast",
+						Precision: 7,
```

Comment:
LOL

Reply:
Yeah, updating the tests annoyed me. :sweat_smile: